meta-dts-distro/recipes-tests/dts-tests/dts-tests/cukinia.conf

```diff
@@ -22,4 +22,6 @@ as "Check if dasharo-hcl-report script exists" cukinia_test -x $(which dasharo-h
 as "Check if txesbmantool binary exists" cukinia_test -x $(which txesbmantool)
 as "Check if smmstoretool package can be started" cukinia_cmd smmstoretool -h
 
+as "Check if cpuid can be started" cukinia_cmd cpuid -v
```

Comment:
Does it work in QEMU to verify if we can return output in a similar format as discussed in the task before ?

```
cpuid -l 0x1a
CPU 0:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Core
CPU 1:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Core
CPU 2:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Core
CPU 3:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Core
CPU 4:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Core
CPU 5:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Core
CPU 6:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Core
CPU 7:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Core
CPU 8:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Atom
CPU 9:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Atom
CPU 10:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Atom
CPU 11:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Atom
CPU 12:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Atom
CPU 13:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Atom
CPU 14:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Atom
CPU 15:
   Native Model ID Information (0x1a/0):
      native model ID of core = 0x1 (1)
      core type               = Intel Atom
```

At least execute it and grep for CPU, or something?

Reply:
Yes, I'm getting output in this form. I changed the test. It passes on QEMU. https://github.com/Dasharo/meta-dts/pull/140/commits/6fdcc9c332e42f9ece861e7ebe88bf6ce8e2b13d#diff-b7ac36ed2f19999098c5dde6bec991415a9c3d894214365e300791ec968e44deR25